src/test/scala/org/graphframes/lib/ConnectedComponentsSuite.scala

```diff
@@ -261,4 +269,52 @@ class ConnectedComponentsSuite extends SparkFunSuite with GraphFrameTestSparkCon
       }.toSet
     assert(actualComponents === expected)
   }
+
+  private def withDisabledAQE(f: => Unit): Unit = {
```

Comment:
https://github.com/graphframes/graphframes/pull/436/commits/30f21e3378921081c3f3aae4d34df7988f27c9e3

Reply:
@rjurney
it seams like its AQE that are making problems..

Reply:
@EnricoMi CC
What do you think?